internal/config/config.go

```diff
@@ -21,20 +22,17 @@ import (
 	"github.com/notaryproject/notation/internal/envelope"
 )
 
-// loadConfigOnce is a function that invokes loadConfig only once.
-var loadConfigOnce = sync.OnceValues(loadConfig)
-
+// LoadConfigOnce is a function that invokes LoadConfig only once.
 // LoadConfigOnce returns the previously read config file.
 // If previous config file does not exist, it reads the config from file
 // or return a default config if not found.
-// The returned config is only suitable for read only scenarios for short-lived processes.
-func LoadConfigOnce() (*config.Config, error) {
-	return loadConfigOnce()
-}
+// The returned config is only suitable for read only scenarios for short-lived
+// processes.
+var LoadConfigOnce = sync.OnceValues(LoadConfig)
```

Comment:
We may need to restore the original implementation as the `LoadConfigOnce` is better to be an immutable function rather than a mutable function pointer.